quartz/plugins/transformers/toc.ts

```diff
@@ -52,7 +52,7 @@ export const TableOfContents: QuartzTransformerPlugin<Partial<Options> | undefin
                 }
               })
 
-              if (toc.length > opts.minEntries) {
+              if (toc.length > 0 && toc.length > opts.minEntries) {
```

Comment:
hmm can you show a screenshot when the toc length is < minEntries. What will happen here?

Reply:
Previously a toc length set to 0 lead to an empty toc when there are no headings on the page:

![image](https://github.com/jackyzha0/quartz/assets/1475672/90e4b938-624a-41e5-b9ef-a651b06ab83d)

Now it's removed instead:

![image](https://github.com/jackyzha0/quartz/assets/1475672/c73e8b5a-fd19-47dc-a5ff-f82f91083402)

Reply:
Ugh then it should behave like this.

Let's maybe do the same as backlinks. we can show that no toc is available for something.

If there is no backlink iirc it will say `no backlinks found`.